apps/starccm/install_starccm.sh

```diff
@@ -3,13 +3,11 @@
 # parameters that can be overridden
 APP_INSTALL_DIR=${APP_INSTALL_DIR:-/apps}
 TMP_DIR=${TMP_DIR:-/mnt/resource}
-STARCCM_INSTALLER_DIR=${STARCCM_INSTALLER_DIR:-/mnt/resource}
+STARCCM_INSTALLER_FILE=${STARCCM_INSTALLER_FILE:-/mnt/resource/STAR-CCM+14.06.004_02_linux-x86_64-2.12_gnu7.1.tar.gz}
```

Comment:
I don't think you should change the environment variable here.  We have just written a blog post using these scripts and it would stop it working.  If you would like to specify an installer you could always add another environment variable, e.g. STARCCM_ARCHIVE_NAME.  The other issue is that the run_case script is based on this installer and so would need to be updated as well for the starccm version

Reply:
ok with this change, i just need this in asap so i can update the blog Paul mentions :-)

Reply:
I think it is best to leave this change and update the blog.